### PR TITLE
Added .vscode setting files for c++ compilation

### PR DESCRIPTION
**Pull request by sandy1618**
This is .vscode setting files which is made in a new branch named sandeep .

### CODE REVIEW

**Comment by codersguild** [comment]
Cool, I shall merge it. 
Suggestion: Never commit any `dotfiles `unless required in the project. Put such files in `.gitignore`.

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "type": "shell",
+            "label": "C/C++: g++ build active file",
+            "command": "/usr/bin/g++",
+            "args": [
+                "-std=gnu++14",
```
Comment (codersguild) [inline .vscode/tasks.json:9]: Will `-std=c++17` also work?

**Review by codersguild — APPROVED** [review]

**Review by codersguild — COMMENTED** [review]
Cool, I shall merge it.
Suggestion: Never commit any `dotfiles `unless required in the project. Put such files in `.gitignore.`

File: .vscode/tasks.json (hunk 1 above)
Comment (sandy1618) [reply .vscode/tasks.json:9]: I am adding update. I see. But for similar environment, I committed this dot file. even this is not allowed?

**Pull request merged**